src/librbd/cache/Utils.cc

```diff
@@ -0,0 +1,23 @@
+// -*- mode:C++; tab-width:8; c-basic-offset:2; indent-tabs-mode:t -*-
+// vim: ts=8 sw=2 smarttab
+
+#include "librbd/cache/pwl/DiscardRequest.h"
```

Comment:
Nit: don't forget to `#include "librbd/cache/Utils.h`

Reply:
Added.